site/_data/config/bk_server.yaml

```diff
@@ -53,6 +82,56 @@ groups:
   - param: numReadWorkerThreads
     description: The umber of threads that handle read requests. If 0, reads are handled by [Netty threads](http://netty.io/wiki/thread-model.html) directly.
     default: 1
+  - param: isForceGCAllowWhenNoSpace
+    description: Whether force compaction is allowed when disk full or almost full Force GC may get some space back, but may also fill up disk space more quickly. This is because new log files are created before GC, while old garbage log files deleted after GC.
```

Comment:
Change this sentence to:

Whether force compaction is allowed when the disk is full or almost full. Forcing GC may get some space back, but may also fill up disk space more quickly. This is because new log files are created before GC, while old garbage log files are deleted after GC.